File: Service/Implement/ThongBaoThanhVienService.cs
namespace Service.Implement
{
    public class ThongBaoThanhVienService : BaseService<ThongBaoThanhVien, IThongBaoThanhVienRepository>
    , IThongBaoThanhVienService
    {
    private readonly IThongBaoThanhVienRepository _ThongBaoThanhVienRepository;
    public ThongBaoThanhVienService(IThongBaoThanhVienRepository ThongBaoThanhVienRepository) : base(ThongBaoThanhVienRepository)
    {
    _ThongBaoThanhVienRepository = ThongBaoThanhVienRepository;
    }
    }
    }

File: Website/src/app/shared/ThongBaoThanhVien.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { ThongBaoThanhVien } from 'src/app/shared/ThongBaoThanhVien.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class ThongBaoThanhVienService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "ThongBaoThanhVien";
    }
}

